src/token.spec.js

```diff
@@ -41,23 +42,24 @@ describe('token', () => {
   // -------------------------------------------------------------------
 
   describe('.delete()', () => {
-    it('deletes a token', async () => {
-      const token = await createToken()
-      const xo2 = await getConnection({credentials: {token}})
-
-      await xo2.call('token.delete', {
-        token
-      })
+    it('deletes a token', () => {
+      defer(async ($defer) => {
+        const token = await createToken()
+        const xo2 = await getConnection({credentials: {token}})
+        $defer(async () => await xo2.close())
```

Comment:
We need to wait the promesse returned by close().
You're right, we don't need  to wait it because the function will be called at the end . and it will return the promess.